pkg/client/k8s/k8s.go

```diff
@@ -143,12 +164,30 @@ func (k *K8sClient) GetService(name string, opts mach_apis_meta_v1.GetOptions) (
 	return sops.Get(name, opts)
 }
 
+// coreV1ServiceOps is a utility function that provides a instance capable of
+// executing various k8s service related operations.
+func (k *K8sClient) coreV1ServiceOps() typed_core_v1.ServiceInterface {
+	return k.cs.CoreV1().Services(k.ns)
+}
+
+// CreateCoreV1Service creates a K8s Service
+func (k *K8sClient) CreateCoreV1Service(svc *api_core_v1.Service) (*api_core_v1.Service, error) {
+	sops := k.coreV1ServiceOps()
+	return sops.Create(svc)
+}
+
 // deploymentOps is a utility function that provides a instance capable of
 // executing various k8s Deployment related operations.
 func (k *K8sClient) deploymentOps() typed_ext_v1beta.DeploymentInterface {
 	return k.cs.ExtensionsV1beta1().Deployments(k.ns)
 }
 
+// extnV1B1DeploymentOps is a utility function that provides a instance capable of
+// executing various k8s Deployment related operations.
+func (k *K8sClient) extnV1B1DeploymentOps() typed_ext_v1beta.DeploymentInterface {
```

Comment:
How is this function different from deploymentOps() ?

Reply:
Yeah it duplicate. I have not marked various methods as deprecated. The duplicates will go away in upcoming PRs.